### DISCUSSION TITLE
Selecting multiple dates in DatePicker component

### DISCUSSION THREAD

**Discussion by feute** [mantinedev/mantine#139] [Q&A] [answered]
Hello! I'm thinking about using this library for a new project that is going to ask for multiple days in a calendar and I was wondering if this was possible with the DatePicker component in some way.

**Comment by rtivital** [comment, marked as answer]
Hi, this feature is not supported natively but you can do a workaround with dayStyles and input format, this is definitely hacky, but should work out fine – https://codesandbox.io/s/eloquent-lake-lizz0?file=/src/App.tsx
Note that there might be some issues in styles as I just wanted to showcase how to implement it, use Styles API to fix those if you decide to use this component

**Reply by feute** [reply]
Thank you! That would work.

**Reply by mohammed-bahumaish** [reply]
thank you :)
